README.md

```diff
@@ -184,9 +184,9 @@ Webhooks endpoints are provided on the following URL:s
 
 | Repo service | Webhook Endpoint URL |
 | ------------- |-------------|
-| GitHub | [ServiceURL]/webhooks/github |
-| GitLab | Not available yet. |
-| BitBucket | Not available yet. |
+| GitHub | [ServiceURL]/hooks/github |
+| GitLab | [ServiceURL]/hooks/gitlab |
+| BitBucket | [ServiceURL]/hooks/bitbucket |
```

Comment:
GitLab and GitHub implementations are not complete, that's why they are marked as "Not available yet". Maybe "Implementation in progress" would be a better description.